File: POO JAVA/Aula11/src/aula11/Pessoa.java
package aula11;

public abstract class Pessoa {
    
    //Atributos da Classe
    protected String nome;
    private int idade;
    private String sexo;

    //Metodos Acessores e Modificadores
    public String getNome() {
        return nome;
    }
    public void setNome(String nome) {
        this.nome = nome;
    }
    public int getIdade() {
        return idade;
    }
    public void setIdade(int idade) {
        this.idade = idade;
    }
    public String getSexo() {
        return sexo;
    }
    public void setSexo(String sexo) {
        this.sexo = sexo;
    }
    
    //Metodo da Classe
    public void fazerAniversario(){
        this.idade++;
    }

    @Override
    public String toString() {
        return "Dados{" + "nome=" + nome + ", idade=" + idade + ", sexo=" + sexo + '}';
    }
    
}

File: POO PHP/Aula11/Pessoa.php
<?php

abstract class Pessoa {

    //Atributos da Classe
    private $nome;
    private $sexo;
    private $idade;
    
    //Metodos Acessores e Modificadores
    public function getNome() {
        return $this->nome;
    }
    public function getSexo() {
        return $this->sexo;
    }
    public function getIdade() {
        return $this->idade;
    }
    public function setNome($nome): void {
        $this->nome = $nome;
    }
    public function setSexo($sexo): void {
        $this->sexo = $sexo;
    }
    public function setIdade($idade): void {
        $this->idade = $idade;
    }

    //Metodo da Classe
    public final function fazerAniversario() {
        $this->idade++;
    }
}
